osu.Game/Screens/Play/Player.cs

```diff
@@ -154,6 +154,10 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
 
+            // BDL load may have aborted early.
+            if (DrawableRuleset == null)
```

Comment:
```suggestion
            if (!LoadedBeatmapSuccessfully)
```
matches another place where this is used:
![CleanShot 2021-05-14 at 10 42 52](https://user-images.githubusercontent.com/22781491/118238406-25180e80-b4a1-11eb-8049-8d3e4ac8e482.png)